tests/api/fixtures.ts

```diff
@@ -128,19 +129,31 @@ type ApiClientFixture = {
 };
 
 export const test = base.extend<ApiClientFixture>({
-  axios: async ({ baseURL }, use) => {
-    const axiosInstance = axios.create({ baseURL });
+  axios: async ({ baseURL, ignoreHTTPSErrors }, use) => {
```

Comment:
I don't have issues with the MR, just trying to understand this part. What does the `rejectUnauthorized` value of the HTTP agent do exactly?

Reply:
if `rejectUnauthorized=true` or not set then default behavior is to validate the certificate that HTTPS uses. If the certificate is not valid then an error will be thrown making our tests unable to be executed. E.g.

Lets imagine we deploy Trustify in a infrastructure that uses Self Signed Certificates then we will see something like the image below: 

![chrome_ssl_01_en](https://github.com/user-attachments/assets/13860c87-4955-4d45-8e9c-c1274943dce8)

rejectUnauthorized=false will make the previous image to by passed and ignored therefore make our tests work if Trustify is deployed using self signed certificates.